portfolio/src/main/webapp/js/main.js

```diff
@@ -23,16 +23,55 @@ $(document).ready(function(){
 })
 
 function getComments() {
-  let max = document.getElementById('num-comments').value;
-  fetch('/list-comments?max=' + max).then(response => response.json()).then((comments) => {
-    const commentsListElement = document.getElementById('comments-container');
-    commentsListElement.innerHTML = '';
-    comments.forEach(comment => {
-      commentsListElement.appendChild(createCommentElement(comment));
+  let numCommentsPerPage = document.getElementById('num-comments').value;
+  let numPages = setupPagination(numCommentsPerPage);
+  numPages.then(() => {
+    fetch('/list-comments?num-comments=' + numCommentsPerPage + '&page-num=' + currentPage())
```

Comment:
You can use template literals here. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Thanks! That is so much cleaner!